webpack/index.d.ts

```diff
@@ -654,7 +654,14 @@ declare namespace webpack {
      */
 
     class BannerPlugin extends Plugin {
-        constructor(banner: any, options: any);
+        constructor(options: {
```

Comment:
`options` is [`optional`](https://github.com/webpack/webpack/blob/8a95bcc2f45c2bfa26ae06e5fbf7a81f6dd37e69/lib/BannerPlugin.js#L24); should be `options?`